lib/honeybadger/json.ex

```diff
@@ -43,7 +43,8 @@ defmodule Honeybadger.JSON do
     |> to_encodeable
   end
 
-  defp to_encodeable(input) when is_pid(input) or is_port(input) or is_reference(input) do
+  defp to_encodeable(input)
```

Comment:
It would be nice to have an example of this (any of these, really) in the json tests directly.

Reply:
I added a test covering this to_encodable function